marconi-mamba/README.md

```diff
@@ -1,78 +1,174 @@
-marconi-mamba
---
 
-This is an initial draft for the Marconi Mamba README file. Going forward, we will need to discuss and refine this document and/or the implementation so that they match.
+# Marconi-Mamba
 
-## Description
-marconi-mamba is a [JSON-RPC](http://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0) HTTP server on top of the [marconi](../marconi/README.md)
+Marconi-Mamba is a lightweight, customizable chain follower application and library for DApp developers who need to index and query the Cardano blockchain. Marconi-Mamba provides a way for DApps to have access to data that has already been stored on-chain by indexing portions of the blockchain.
 
-## Playing with the JSON-RPC
-There is an example client/server JSON-RPC to experiment with the JSON-RPC clien/server. Here are the steps to build and execute the examples:
+## Purpose
+
+The purpose of Marconi-Mamba is to make the core Marconi APIs available to non-Haskell applications.
+
+## Interface
+
+The interface for Marconi-Mamba uses [JSON-RPC](http://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0) (JSON remote-procedure calls) over HTTP built on top of [Marconi](../marconi/README.md).
```

Comment:
is it required to put `(JSON remote-procedure calls)`?
Since the audience is assumed to be developers + the link, this may be an overkill?